bindings/wasm/manifold.d.ts

```diff
@@ -299,7 +299,7 @@ declare class Manifold {
  * @param size The X, Y, and Z dimensions of the box.
  * @param center Set to true to shift the center to the origin.
  */
-declare function cube(size?: Vec3|number, center?: boolean): Manifold;
```

Comment:
This makes sense as a change for our library, but it broke manifoldCAD.org which also loads the functions into the global namespace. I'm curious what the cleanest way to fix that is in our editor.d.ts file?

Reply:
See #344 - would be nice to find a way to improve this.